include/cameras/CameraBase.h

```diff
@@ -46,35 +50,50 @@ class CameraBase
   void start();
 
   /**
-   * @brief Has to be overridden. Executes once all the steps that the camera implements.
+   * @brief Executes once all the steps that the camera implements.
    */
-  virtual void run_pipeline() = 0;
+  void run_pipeline();
 
   /**
-   * @brief Has to be overridden. Polls camera for a frame + eventually extra processing.
+   * @brief Polls the camera until the buffer is empty. ensuring the frame is the most recent one.
    */
-  virtual void poll() = 0;
+  bool poll();
```

Comment:
No need to return bool from poll(), it never returns false.

Reply:
Indeed, removed.